bip-0009.mediawiki

```diff
@@ -229,6 +235,10 @@ upgrades for successful soft forks.
 
```

Comment:
The threshold needs to be changed as well see https://github.com/bitcoin/bitcoin/pull/21377#discussion_r611493576

Reply:
It's not clear to me if the speedy trial permanently reduces the threshold, or if this was just a lazy way to set it to 90% for the trial itself. BIP 9 already allows for a lower threshold.